pyopencl/invoker.py

```diff
@@ -313,8 +313,8 @@ def _get_max_parameter_size(dev):
     from pyopencl.characterize import get_pocl_version
 
     dev_limit = dev.max_parameter_size
-
-    if get_pocl_version(dev.platform) is not None:
+    pocl_version = get_pocl_version(dev.platform, fallback_value=(1, 8))
+    if pocl_version is not None and pocl_version < (3, 0):
```

Comment:
FYI @matthiasdiener This should've been here from the get-go. I didn't catch this in review.

Reply:
I see, thanks! The reason I had written https://github.com/inducer/pyopencl/pull/555 in the previous way was that https://github.com/pocl/pocl/pull/1046 hadn't been merged at that point, and so it wasn't clear which (if any) pocl version would fix the value.

Reply:
Sure, that makes sense. I'm just wary of having workarounds without expiration dates.